app/assets/stylesheets/pages/_home.scss

```diff
@@ -15,3 +20,11 @@
     border-radius: 0;
   }
 }
+
+h4 {
+  font-family: 'Recursive', sans-serif;
```

Comment:
Can use `$subtitle-font` here instead of `"Recursive", sans-serif;` if you put it into a variable in the _fonts.scss config